codebreaker_smn/lib/codebreaker_smn/helpers/validation_helper.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Validate parameters
+module ValidationHelper
```

Comment:
why you have two such helpers?

Reply:
Kinda' validations on both sides - client and server